src/components/templates/pipelines.ts

```diff
@@ -0,0 +1,76 @@
+export const newPipelineTemplate = `
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: new-pipeline
+spec:
+  params:
+    - name: paramName
+      type: string
+  resources:
+    - name: app-git
+      type: git
+    - name: app-image
+      type: image
+  tasks:
+    - name: build-app
+      taskRef:
+        name: s2i-java-11
+        kind: ClusterTask
+      resources:
+        inputs:
+        - name: source
+          resource: app-git
+        outputs:
+        - name: image
+          resource: app-image
+`;
+
+export const newPipelineResourceTemplate = `
+apiVersion: tekton.dev/v1
+kind: PipelineResource
+metadata:
+  name: nginx-ex-git-resource
+spec:
+  type: git
+  params:
+    - name: url
+      value: https://github.com/sclorg/nginx-ex.git
+    - name: revision
+      value: master
+`;
+
+export const newTaskTemplate = `
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: example-task
+spec:
+  params:
+    - name: appName
+      type: string
+  steps:
+  - image: registry.redhat.io/ubi7/ubi-minimal
+    command:
+    - /bin/bash
+    - '-c'
+    - echo
+    - $(inputs.params.appName)
+`;
+
+export const newTaskRunTemplate = `
```

Comment:
I do not see this template when creating a new TaskRun and same happening with new Pipeline.

Reply:
For TaskRun I resolved, for Pipeline due to /builder page, this default template is not adding, If I remove /builder from URL i can see the template.